app/javascript/components/upload/UploadWizard.jsx

```diff
@@ -135,6 +136,10 @@ export function RawUploadWizard({ studyAccession, name }) {
     })
   }
 
+  if (!window.SCP.readOnlyToken) {
+    fetchReadOnlyToken(studyAccession)
+  }
```

Comment:
See comment beginning "Do callers of this need".  I suspect we'd want to call `setupRenewalForReadOnlyToken` (with the slight extension noted there) unconditionally here.

```suggestion
  setupRenewalForReadOnlyToken(studyAccession)
```

Reply:
[a6ff9d9](https://github.com/broadinstitute/single_cell_portal_core/pull/1963/commits/a6ff9d91c1e2f85f97f3c250c3d107772e62262e)